### PR TITLE
7132 - Health check on pipeline

### PR DESCRIPTION
**Pull request by ravi102465**
This PR has changes for 
 Adding Health check step on pipeline and fail pipeline if deployment is not successfully.

### CODE REVIEW

File: Deployment/check_service_status.ps1 [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    
+        If ($HttpStatus -eq 200 ) {
+            Write-Host "Service is up. Stopping Polling ..."
+            $isServiceActive = 'true'
+            break
+        }
+        Else {
+            Write-Host "Service not yet Up. Status code: $HttpStatus ..."
+            Start-Sleep -Seconds $sleepTimeInSecond
+        }
```
Comment (vijendr13545) [inline Deployment/check_service_status.ps1:31]: It looks sleep block is repeated and code will wait for twice of `sleepTimeInSecond`. Please can you check.

```suggestion
        $HttpRequest  = [System.Net.WebRequest]::Create("$healthEndPointUrl")
        $HttpResponse = $HttpRequest.GetResponse() 
        $HttpStatus   = $HttpResponse.StatusCode
        Write-Host "Status code of web is $HttpStatus ..."
    
        If ($HttpStatus -eq 200 ) {
            Write-Host "Service is up. Stopping Polling ..."
            $isServiceActive = 'true'
            break
        }
        Else 
        {
            Write-Host "Service not yet Up. Status code: $HttpStatus ..."
         }
        Start-Sleep -Seconds $sleepTimeInSecond
```

Comment (ravi102465) [reply Deployment/check_service_status.ps1]: Removed sleep from all places and added at the end for loop.

**Review by vijendr13545 — APPROVED** [review]
Looks good to me.

**Review by rockydevnet — APPROVED** [review]
LGTM

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**